core/source/Http.mint

```diff
@@ -15,12 +15,23 @@ record Http.Request {
 
 /* Represents an HTTP response. */
 record Http.Response {
-  status : Number,
-  body : String
+  headers : Map(String, String),
+  body : Http.ResponseBody,
+  bodyString : String,
+  status : Number
+}
+
+/* Represents the body of an HTTP response. */
+enum Http.ResponseBody {
+  Json(Object)
+  Html(String)
+  Text(String)
+  File(File)
 }
```

Comment:
We could also handle XML (via [`DOMParser`](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser))

```suggestion
enum Http.ResponseBody {
  Json(Object)
  XML(Object)
  Html(String)
  Text(String)
  File(File)
}
```